### PR TITLE
Chef provisioning, general update

### LINKED ISSUE #21: trusty64
Migrate the virtual machine to trusty64 image. Maybe it will be more stable.

### PR DESCRIPTION
**Pull request by ForNeVeR**
I've implemented a simple Chef wrapper for both horta and horta-web. There is still some work for horta-web automating but I will create a separate issue for it.

This pull request will close #21 and #22 (because that was the purpose), #20 because #22 has superseded it, #13, #14 and #15 because it deprecates code compilation on the deployment box, so `sbt` issues are no more.

### CODE REVIEW

**Comment by ForNeVeR** [comment]
cc @rexim because he's known as a reputed Ruby coder

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,8 @@
 /.vagrant/
+/Berksfile.lock
```
Comment (rexim) [inline .gitignore]: ERMAHGERD!!

File: README.md [hunk 2]
```diff
@@ -13,17 +13,37 @@ Install the Vagrant package for your operation system. By default you will also
 [VirtualBox](https://www.virtualbox.org/). VirtualBox is just the default option; there is nothing in the
 horta-foundation that has strict dependency on VirtualBox.
 
-Starting the virtual machine
-----------------------------
-It is as simple as
+Configuring the Vagrant
+-----------------------
+Ypu may want to change Vagrant default forwarded SSH port. To do that, edit the `Vagrantfile`.
```
Comment (rexim) [inline README.md]: > Ypu

File: README.md [hunk 3]
```diff
@@ -13,17 +13,37 @@ Install the Vagrant package for your operation system. By default you will also
...
+
+Install the `trusty64` image if yo have not already done it:
```
Comment (rexim) [inline README.md]: >  yo

**Comment by rexim** [comment]
Consider the shit approved after above remarks are fixed.

File: README.md (hunk 3 above)
Comment (ForNeVeR) [inline README.md]: That one is right I think.

Comment (rexim) [inline README.md]: Well, ok then

**Pull request merged**